README.md

```diff
@@ -97,6 +97,43 @@ Other available operations may be discovered by running:
 ./project.rb
 ```
 
+#### API: Faster API startup for MacOS
+The above steps for starting the API server can take upwards of 8-10 minutes on MacOS, most likely due to performance issues with Docker for Mac. Follow these steps to set up your developer environment to start the API server outside of docker. A full restart should take ~30 seconds with this method.
+
+All commands should be run from `workbench/api`
+
+##### Setup
+* Go through the [normal docker developer startup process](#api-dev-appengine-appserver) at least once successfully
+* Install Java 8 and set JAVA_HOME. Note: Your Java8 library directory may be different
+    * ```Shell
+      export JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk1.8.0_201.jdk/Contents/Home
+      ```
+    * Optional - Add the export command to your `.bash_profile` or `.bashrc` and `source` it 
+* Install envsubst
+    * ```Shell
+      brew install gettext
+      brew link --force gettext
+      ```
+* Install Java App Engine components
+    * ```Shell
+      gcloud components install app-engine-java
+      ```
+* Generate Google Cloud access token
+    * ```Shell
+      ./project.rb get-test-service-creds
+      ```
+
+##### Usage
+* Start services required for API server
+    * ```Shell
+      ./project-rb start-api-reqs
+      # the counterpart command is ./project.rb stop-api-reqs
+      ```
+* Start API server through gradle
+    * ```Shell
+      ./gradlew appengineRun
```

Comment:
An alternative approach would be to keep everything else the same, and just switch `run-api` to do this instead.

Reply:
I'm not sure about updating `./project-rb run-api` since I believe it can be confusing if `./project-rb` starts to mix dockerized and non dockerized setups.

So I think it should stay as is or we can just remove it entirely if it is not being used. I'm planning to add some usage metrics around `./project-rb` to track how often developers are using the various commands so we can do some cleanup of older/unused code.

Reply:
The original intent with ./project.rb was to be the location for developer commands in lieu of the more typical ad-hoc shell scripting. It's not directly related to docker usage, though it may have developed that association over time (there are already commands in there that do not use docker).

Putting all of these things into one command seems a bit easier to me, but if that would cause a significant slowdown then this is fine as-is I guess, e.g.:

```
get-test-service-creds
start-api-reqs
begin:
  run ./gradlew ....
ensure:
  stop-api-reqs
```